drivers/kscan/kscan_mchp_xec.c

```diff
@@ -33,6 +36,16 @@ LOG_MODULE_REGISTER(kscan_mchp_xec);
 /* Thread stack size */
 #define TASK_STACK_SIZE 1024
 
+struct kscan_xec_config {
+	uintptr_t regbase;
```

Comment:
You can store this directly:

```suggestion
	struct kscan_regs *regs;
```

Reply:
Fixed in latest push